nsv13/code/game/gamemodes/pvp/pvp.dm

```diff
@@ -0,0 +1,237 @@
+/**
+
+Shamelessly ripped off from nuclear.dm
+
+*/
+
+GLOBAL_LIST_EMPTY(syndi_crew_spawns)
+
+GLOBAL_LIST_EMPTY(syndi_crew_leader_spawns)
+
+/datum/game_mode/pvp
+	name = "PVP"
+	config_tag = "pvp"
+	report_type = "nuclear"
+	false_report_weight = 10
+	required_players = 30 // 30 players initially, with 15 crewing the hammurabi and 15 crewing the larger, more powerful hammerhead
+	required_enemies = 10
+	recommended_enemies = 10
+	antag_flag = ROLE_SYNDI_CREW
+	enemy_minimum_age = 0
+
+	announce_span = "danger"
+	announce_text = "Nanotrasen's incursions into Syndicate space have not gone unnoticed!\n\
+	<span class='danger'>Operatives</span>: Eliminate Nanotrasen's presence in Syndicate space using the weaponry of the SSV Hammurabi, or via a thermonuclear detonation.\n\
+	<span class='notice'>Crew</span>: Survive the Syndicate assault long enough for reinforcements to arrive."
+
+	title_icon = "nukeops"
+
+	var/list/pre_nukeops = list()
+
+	var/nukes_left = 1
+
+	var/datum/team/nuclear/nuke_team
+	var/highpop_threshold = 45 //At what player count does the round enter "highpop" mode, and spawn the Syndicate a larger ship to compensate.
+
+	var/operative_antag_datum_type = /datum/antagonist/nukeop/syndi_crew
+	var/leader_antag_datum_type = /datum/antagonist/nukeop/leader/syndi_crew
+	var/time_limit
+	var/list/standard_ships = list("Hammurabi.dmm") //Update this list if you make more PVP ships :) ~Kmc
+	var/list/highpop_ships = list("Hulk.dmm") //Update this list if you make a big PVP ship
+
+/**
+
+Method to spawn in the Syndi ship on a brand new Z-level with the "boardable" trait active so we can fly to it.
+
+*/
+
+/datum/game_mode/pvp/pre_setup()
+	var/pop = num_players()
+	var/enemies_to_spawn = required_enemies + round((pop-required_enemies)/10) //Syndicates scale with pop. On a standard 30 pop, this'll be 30 - 10 -> 20 / 10 -> 2 floored = 2, where FLOOR rounds the number to a whole number.
+	var/map_file = pop < highpop_threshold ? pick(standard_ships) : pick(highpop_ships) //Scale the ship map to suit player pop. Larger crews need more space.
+	var/ship_type = pop < highpop_threshold ? /obj/structure/overmap/syndicate/pvp : /obj/structure/overmap/syndicate/pvp/hulk
+	var/obj/structure/overmap/syndiship
+	for(var/z in SSmapping.levels_by_trait(ZTRAIT_CORVI))
+		syndiship = new ship_type(get_turf(locate(round(world.maxx * 0.5, 1), round(world.maxy * 0.5, 1), z))) //Make a new syndie ship object in Corvi.
+	message_admins("Spawning in syndi ship map, this may take a while. No the game hasn't crashed, I'm just loading a map before we start.") //Warn the admins. This shit takes a while.
+	var/list/errorList = list()
+	var/list/loaded = SSmapping.LoadGroup(errorList, "Syndicate ship", "map_files/PVP", map_file, default_traits = ZTRAITS_MINING_SHIP, silent = FALSE)
```

Comment:
If the event isn't disabled, won't making it the "mining ship" occasionally spawn belt rats on top of them?

Reply:
The "Mining ship" ZTRAITS are just to allow it to be boardable, self looped space etc. We ought to actually refactor this to ZTRAITS_BOARADABLE_SHIP or such

Reply:
Then that has to be done before this is merged.